src/components/formSections/components/TermsListField.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```

Comment:
I may have misguided you here, but could you please use _IntlConsumer_ instead of
 _injectIntl_ like how we do it here? https://github.com/folio-org/stripes-erm-components/blob/master/lib/DocumentsFieldArray/FileUploaderField.js#L4 because we have decided to not use _ injectIntl_ until the _react-intl 3.0_ version is released which is now still in the beta version